doc/nrf/protocols/matter/getting_started/custom_clusters.rst

```diff
@@ -153,6 +153,8 @@ See the description of each element in the following tabs:
          * ``source`` - Specifies whether the event originates from the client or server.
          * ``code`` - A unique identifier for the event within the cluster.
          * ``name`` - The name of the event.
+         * ``priority`` - The priority of the event.
+                          The valid values are 'debug', 'info', and 'critical'.
```

Comment:
```suggestion
                          The valid values are 'debug', 'info', and 'critical'.
```
Should these be between double backticks. Or is that single quote character needed here?

Reply:
I think that double backticks should be good, thanks for the catch.